libs/postgres_versioninfo/src/lib.rs

```diff
@@ -113,10 +113,10 @@ impl PgMajorVersion {
 impl Display for PgMajorVersion {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         f.write_str(match self {
-            PgMajorVersion::PG14 => "PgMajorVersion::PG14",
-            PgMajorVersion::PG15 => "PgMajorVersion::PG15",
-            PgMajorVersion::PG16 => "PgMajorVersion::PG16",
-            PgMajorVersion::PG17 => "PgMajorVersion::PG17",
+            PgMajorVersion::PG14 => "14",
+            PgMajorVersion::PG15 => "15",
+            PgMajorVersion::PG16 => "16",
+            PgMajorVersion::PG17 => "17",
```

Comment:
I don't like using plain numbers for something that is more than just that plain number, unless context makes it abundantly clear. As Display doesn't guarantee context, this was added by adding the full enum tag to Display output.

Instead of neutering the Display implementation, I think it's better to fix the clap usage so that it doesn't rely on `Display` for serializing strings - we implement Serialize and Deserialize for those purposes.